datascience/table.py

```diff
@@ -749,17 +757,22 @@ def wrapped(column):
     return wrapped
 
 
-def _fill_with_zeroes(order, rows, zero=None):
-    """Return a column of the index-1 elements in rows, where the index
-    of each value is determined by matching index-0 to an element of keys.
+def _fill_with_zeroes(keys, rows, zero=None):
+    """Return a column of the index-(-1) elements in rows, where the index
```

Comment:
I have a really hard time figuring out what this function does. Now seems like a good time to clean up the description or reduce its complexity in some way. Like, what's `index`?